chaos-duck/chaos_duck/__main__.py

```diff
@@ -23,8 +22,9 @@ async def main(iterations: int):
 
     task_runs: dict[UUID, int] = {}
     for i in range(iterations):
+        await asyncio.sleep(0)
```

Comment:
What's this for?

Reply:
Ah yes, since submission is sync now, there was no yielding at all during this whole submission phase (this loop blocks the event loop), so the chaos agents would never get a chance to wreak havoc during submission.